nasa-gibs/worldview
# WV-2985: Removing Date for EIC Static Map Mode

## Description

This PR removes the date in EIC mode when the static map is displayed. 

## How To Test

1. `git checkout wv-2985`
2. `npm ci`
3. `npm run watch`
4. I have configured the `CERES_EBAF_TOA_Shortwave_Flux_All_Sky_Monthly` layer for static map testing in EIC mode since it has not had coverage for several years. Use this to load EIC mode with this layer. http://localhost:3000/?v=-181.74360912131363,-98.53068072538338,182.80846643543055,106.52986177528524&df=true&kiosk=true&eic=si&l=CERES_EBAF_TOA_Shortwave_Flux_All_Sky_Monthly&lg=false
5. Give WV 3 to 5 seconds to cycle the API calls and then verify that the Blue Marble static map loads without the date in the bottom left hand corner.
6. Load a normal EIC scenario such as http://localhost:3000/?v=-181.74360912131363,-98.53068072538338,182.80846643543055,106.52986177528524&df=true&kiosk=true&eic=si&l=Coastlines_15m(opacity=0.71),GHRSST_L4_MUR_Sea_Surface_Temperature(palette=divergent_1)&lg=true
7. Verify that the date remains visible. 

@nasa-gibs/worldview
